app/controllers/users_controller.rb

```diff
@@ -19,8 +27,47 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
+  end
+
+  def update
+    @user = User.find_by(id: params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = "Profile updated"
+      redirect_to @user
+    else
+      render "edit"
```

Comment:
```suggestion
      render :edit
``` 
and add flash for this case